java/kotlin-extractor/build.py

```diff
@@ -169,7 +169,7 @@ def compile(jars, java_jars, dependency_folder, transform_to_embeddable, output,
         shutil.rmtree(tmp_src_dir)
     shutil.copytree('src', tmp_src_dir)
 
-    include_version_folder = tmp_src_dir + '/main/kotlin/utils/versions/to_include'
+    include_version_folder = tmp_src_dir + '/main/kotlin/utils/this_version'
```

Comment:
I don't understand these build.py changes.

Reply:
Before, we removed directories like `versions/1.2.3` where `1.2.3` is a version we know about. This would leave just `versions/to_include`, which is the files that we want for the version that we are compiling for.

In this PR, I add a `1.9.255` directory that we don't know about, containing files we'll need in the future. With the old code, that directory wouldn't be removed, and so we would try to compile it for all Kotlin versions, which goes wrong.

So now, we remove the entire `versions/` directory. This means that we remove support for all versions, even those that we don't know about yet. But doing this requires moving `versions/to_include` and `Psi2IrFacade` out of `versions/`, so that they don't get removed.

Reply:
Thanks for the explanation.